### DISCUSSION TITLE
Embed Player not Displaying Correctly

### DISCUSSION THREAD

**Discussion by gulicny** [AzuraCast/AzuraCast#6013] [General]
Hey all, I am having an issue adding embedding the iframe player to my site. Despite trying to add the custom CSS into the admin/branding area, I am still not seeing the text color or background color change. I apologize if I am missing something simple here, I am still a novice.

Here is my CSS:

```
[data-theme] body.station-stupid_action_online .public-page .card  {
    background-color: #77ff00 !important;
    color: rgb(0, 0, 0) !important;
  	
}

[data-theme] body.station-stupid_action_online .public-page .card  {
    background-color: #f0afaf !important; 
  	color: rgb(0, 0, 0) !important;
}
```

![Screen Shot 2023-01-11 at 7 26 32 PM](https://user-images.githubusercontent.com/43893996/211947818-a8a84897-4aa0-4085-b877-24539932ae50.png)

**Comment by guiltlab** [comment]
Hey, did you have a look at the documentation ?
https://docs.azuracast.com/en/administration/customization/css

There are some examples that might help there. You can also find some examples here: https://github.com/TychoWerner/AzuraCastMods

Hope that helps.

**Reply by gulicny** [reply]
I have pulled the CSS directly from those examples and it is still displaying incorrectly. Thank you though :)
